### ISSUE TITLE
Single-line @param/@return defines should ignore leading hyphen in description

### ISSUE THREAD

**Issue by nantas** [fireball-x/firedoc#3]
In the following property docs:

``` js
    /**
     * Get the local matrix that transforms a point from local space into parents space.
     * @method getLocalMatrix
     * @param {Matrix23} [out] - optional, the receiving vector
     * @return {Matrix23}
     */
```

will produce an unnecessary list item glyph like in markdown list notion:

```
- list item
```

The hyphen's purpose is to separate parameter name and description, makes the source code more readable. It should not be processed as a markdown list item notion.

**Comment by nantas** [comment]
This also happens in @return tags:

![image](https://cloud.githubusercontent.com/assets/344547/7646774/f2a8fb82-fafa-11e4-90e6-ad75803f7878.png)

**Comment by yorkie** [comment]
Okay, fixed also for `@return` at e36b463e2481a577b84acea2689321a2da419c87, by hand I added multi-language support, though.
